_posts/2022-11-03-what-is-new-in-strimzi-0.32.0.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title:  "Video: What's new in Strimzi 0.32.0"
+date: 2022-11-03
+author: jakub_scholz
+---
+
+Strimzi 0.32.0 has been released with multiple new features and improvements.
+As with previous releases, we created a video which introduces the main changes.
+You can watch the video here or in our YouTube channel.
+
+<!--more-->
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/q4TXUVePRJ8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
adding this post was on purpose? I mean to include in this PR instead of having a new one.

Reply:
Its on purpose. Its part of the release. The reason why I did not add it in one Pr in previous releases was that I forgot about it always.

Do you see any reasons why it should not be included and should have a separate Pr instead?

Reply:
It is just that I saw the blog post as separated from the operator doc, but it's not a problem at all, no concerns.